src/iccpd/src/iccp_ifm.c

```diff
@@ -948,17 +952,29 @@ void iccp_from_netlink_port_state_handler( char * ifname, int state)
             if (lif_po->type == IF_T_PORT_CHANNEL && strncmp(lif_po->name, ifname, MAX_L_PORT_NAME) == 0)
             {
                 mlacp_portchannel_state_handler(csm, lif_po, po_is_active);
+                is_mclag_intf = 1;
             }
         }
+        if (!is_mclag_intf)
+       {
+           lif_po = local_if_find_by_name(ifname);
+           update_orphan_port_mac(csm, lif_po, po_is_active);
+       }
+
     }
 
     return;
 }
 
+//this function is no more required - vlan membership updates comes through
+//state db updates from mclagsyncd
```

Comment:
Are the changes in this function needed?

Reply:
@prvattem: Thanks for the comment. The unused function iccp_parse_if_vlan_info_from_netlink is being removed.